akka-http-core/src/main/scala/akka/http/impl/engine/server/HttpServerBluePrint.scala

```diff
@@ -461,6 +461,12 @@ private[http] object HttpServerBluePrint {
       def emitErrorResponse(response: HttpResponse): Unit =
         emit(responseCtxOut, ResponseRenderingContext(response, closeRequested = true), () ⇒ completeStage())
 
+      def maybePullRequestParsingIn(): Unit =
```

Comment:
Since pull-suppression is related to `oneHundredContinueResponsePending` wouldn't it be more appropriate to make that tri-state (since I assume that oneHundredContinueResponsePending cannot be false while pullSuppressed is true)?